testdata/goldens/existing_test_file.go

```diff
@@ -45,6 +46,14 @@ func TestBaz100(t *testing.T) {
 	}
 }
 
+func wrapToString(in []int) []string {
```

Comment:
I think you can revert the changes in this file.

Reply:
And for which function should I generate test?